packages/console/src/components/Button/index.module.scss

```diff
@@ -70,7 +95,7 @@
       border-color: var(--color-neutral-70);
     }
 
-    &:focus {
+    &:focus-visible {
```

Comment:
would. 
```
&:focus,
&:focus-visible
```

better?

Reply:
No, we should not add `outline` to `:focus` pseudo class, as the outline effect is only needed when pressing tab key, not on mouse click.
If we add outline to `:focus` as well, mouse click will also have the outline effect, which is not what we want.